watch-parties/src/main/webapp/videoPlayer.js

```diff
@@ -0,0 +1,101 @@
+// Loads in the iFrame Player API code asynchronously
+var tag = document.createElement('script');
+
+tag.src = "https://www.youtube.com/iframe_api";
+
+var firstScriptTag = document.getElementsByTagName('script')[0];
+firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+
+// Create the player object
+var player;
+
+function onYouTubeIframeAPIReady() {
+    player = new YT.Player('player', {
+        height: '390',
+        width: '640',
+        videoId: 'QSQwZlRMVAM',
+        events: {
+        // Adds event listeners for the player and maps the function that will triggered
+        // See Events docs for player event listeners:
+        // https://developers.google.com/youtube/iframe_api_reference#Events
+
+        'onReady': onPlayerReady,
+        'onStateChange': onPlayerStateChange
+        }
+        });
+}
+
+// When the player state is changed, the API calls this function
+function onPlayerStateChange(event) {
+    var status = player.getPlayerState();
+
+    $.ajax({
+        url: 'sync',
+        method: 'POST',
+        data: {status : status},
+        success    : function(resultText){
+            $('#result').html(resultText);
+            setTimeout(() => { console.log("Changed playback state"); }, 1000);
```

Comment:
Could you add constants for the timeout value here and below (line 62) also? Since they are the same you can make the constant file scoped and at the top of the file.